### PR TITLE
Add support of python 3.13 and django 5.1

### PR DESCRIPTION
**Pull request by jcapiitao**

### CODE REVIEW

**Comment by jcapiitao** [comment]
If this PR is merged, could you issue a new release on PyPI, I'd like to package it in Fedora.
There are chance that this project will be used by the OpenStack project [1]

[1] https://review.opendev.org/c/openstack/requirements/+/934220

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -12,20 +12,8 @@ jobs:
...
-          - django-version: 32
-            python-version: 3.12
-          - django-version: 41
-            python-version: 3.12
-          - django-version: 50
-            python-version: 3.8
-          - django-version: 50
-            python-version: 3.9
+        django-version: [50]
+        python-version: ["3.10", 3.11, 3.12]
```
Comment (gasman) [inline .github/workflows/test.yml]: Let's leave Django 4.1 and Python 3.9 in here, as these are still in security / LTS support.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:  # https://docs.djangoproject.com/en/5.0/faq/install/#faq-python-version-support
         django-version: [50]
-        python-version: ["3.10", 3.11, 3.12]
+        python-version: [3.10, 3.11, 3.12]
```
Comment (gasman) [inline .github/workflows/test.yml:16]: Are you sure these are unneeded? We've had issues with yaml treating 3.10 as 3.1 before.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -12,9 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:  # https://docs.djangoproject.com/en/5.0/faq/install/#faq-python-version-support
-        django-version: [50]
+        django-version: [50, 51]
         python-version: [3.10, 3.11, 3.12]
```
Comment (gasman) [inline .github/workflows/test.yml:16]: Need 3.13 in here, I think...?

**Review by gasman — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (jcapiitao) [reply .github/workflows/test.yml]: I double checked and looks like it's 4.2 the LTS one according to https://www.djangoproject.com/download/
I readded it with py3.9 as minimal python runtime

File: .github/workflows/test.yml (hunk 2 above)
Comment (jcapiitao) [reply .github/workflows/test.yml]: Not 100% sure, but I checked other projects that don't use double quote and it's running fine,

File: .github/workflows/test.yml (hunk 3 above)
Comment (jcapiitao) [reply .github/workflows/test.yml]: Right, got blurred by the exclude list ..

**Comment by yeroshalmimLF** [comment]
BUMP! We need this as well please @gasman

**Comment by gasman** [comment]
@yeroshalmimLF Do you, though...? These are only changes to the testing infrastructure - there's nothing stopping you from running the current version of django-libsass against the latest Django/Python, as far as I'm aware.

File: .github/workflows/test.yml (hunk 1 above)
Comment (gasman) [reply .github/workflows/test.yml]: Ah yep, Django 4.2 is the one I meant, sorry :-)

File: .github/workflows/test.yml (hunk 2 above)
Comment (gasman) [reply .github/workflows/test.yml]: I just set CI running for this PR, and it's trying to run against 3.1, so it looks like this is needed...

I'd suggest adding the quotes around all python versions here, for consistency.

Comment (jcapiitao) [reply .github/workflows/test.yml]: Weird, I added the quotes :+1:

**Comment by gasman** [comment]
Thanks @jcapiitao!

**Pull request merged**